kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -4,7 +4,7 @@
     :appBarTitle="learnString('exploreLibraries')"
     :route="back"
     :primary="false"
-    :loading="loading"
+    :loading="loading.value"
```

Comment:
This shouldn't be necessary to access the value in a template. If you are having to do this, something has gone awry.